### PR TITLE
Fix create button

### PR DESCRIPTION
**Pull request by neelxie**
#### What is the PR about?
The PR fixes the create button

image

> After
image

### CODE REVIEW

File: src/components/ButtonComponent/ButtonComponent.css [hunk 1]
```diff
@@ -4,13 +4,17 @@
...
 
 
-.RoundAddButton:focus {
+.RoundAddButton:hover {
     box-shadow: none;
     border-color:transparent!important;
     outline:none;
 }
+
+.RoundAddButton svg{
```
Comment (mrwagaba) [inline src/components/ButtonComponent/ButtonComponent.css:18]: simply make `.RoundAddButton` a flex item and set `justify-content` and `align-items` to `center`

Comment (StephenAraka) [reply src/components/ButtonComponent/ButtonComponent.css:18]: ...that and remove the !important

Comment (neelxie) [reply src/components/ButtonComponent/ButtonComponent.css]: @StephenAraka @mrwagaba this has been worked on.

**Comment by neelxie** [comment]
> @neelxie just the above fixes and your good

@MutegekiHenry done sir.

**Pull request merged**